### ISSUE TITLE
Existing metric as a template for a new one

### ISSUE THREAD

**Issue by vkaverin** [VKCOM/statshouse#706]
### User story
Given existing metric `A` with many tags.  Want to create new metric `B` which will have the same tags as `A`, but with some changes.

Currently I have to type all the tags for `B` manually.

### Desired solution
A button on metric `A` like `New metric from this one` which will create new metric `B` with tags from `A`.

Probably it depends on #194.

**Comment by shtativ** [comment]
Ticket: SH-90

**Comment by hiSandog** [comment]
The scope would be easier to review if the first version defined a small acceptance contract for `Existing metric as a template for a new one`. I would call out the default behavior, the migration/backwards-compatibility story, and one or two cases that are intentionally out of scope so the implementation does not grow beyond the original request.
